public/controllers/userController.js

```diff
@@ -69,28 +104,63 @@ angular.module('userController', ['userRegService'])
         locRegData.phone_no   == "" || locRegData.phone_no   == undefined ||
         locRegData.password   == "" || locRegData.password   == undefined ||
         locRegData.retypepass == "" || locRegData.retypepass == undefined) {
-        console.log("some details required!")
+      
+        console.log(_this.regData)
+
+        if (_this.regData == undefined  || locRegData.username == "" || locRegData.username   == undefined) {
+          _this.inputFields.username = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.fullname   == "" || locRegData.fullname   == undefined) {
+          _this.inputFields.fullname = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.emailid    == "" || locRegData.emailid    == undefined) {
+          _this.inputFields.emailid = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.country    == "" || locRegData.country    == undefined) {
+          _this.inputFields.country = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.address    == "" || locRegData.address    == undefined) {
+          _this.inputFields.address = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.phone_no   == "" || locRegData.phone_no   == undefined) {
+          _this.inputFields.phone_no = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.password   == "" || locRegData.password   == undefined) {
+          _this.inputFields.password = "emptyFields"
+          _this.inputFields.password = "emptyFields"
+        }
+        if (_this.regData == undefined  || locRegData.retypepass == "" || locRegData.retypepass == undefined) {
+          _this.inputFields.retypepass = "emptyFields"
+        }
         return false;
```

Comment:
Use for each loop to iterate through all keys in the object, since same function is being applied to all keys in the object